src/OpalCompiler-Core/RBPragmaNode.extension.st

```diff
@@ -6,7 +6,7 @@ RBPragmaNode >> asIRPrimitive [
 	| args module name spec |
 	args := (self arguments collect: [ :each | each value ]) asArray.
 	"normal methods have primitive number 0"
-	self isPrimitive ifFalse: [ IRPrimitive null ].
+	self isPrimitive ifFalse: [ ^ IRPrimitive null ].
```

Comment:
Maybe I am losing something, but this is not the same behavior.

Reply:
yes, the old version was a NOP. (as  IRPrimitive null has no effect).

The only client does this check itself, this is why it was never a problem.

(yes, maybe we should do a test...)